yaml-sort.js

```diff
@@ -88,28 +93,56 @@ argv.input.forEach((file) => {
     }
 
     const output =
-        argv.stdout || (argv.output === '.') || (isStdin && !argv.output)
-          ? process.stdout.fd
-          : (argv.output ? argv.output : file)
+      argv.stdout || (argv.output === '.') || (isStdin && !argv.output)
+        ? process.stdout.fd
+        : (argv.output ? argv.output : file)
 
     const content = fs.readFileSync(isStdin ? process.stdin.fd : file, argv.encoding)
+    let documents
+    try {
```

Comment:
@Romain7495 This code is already within a try/catch.

Reply:
totally agree sorry, I removed it, but I have added a test case for duplicated keys